### ISSUE TITLE
Sitemaps should expose all links for each site

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4231]
**User stories**
As a user of the sitemaps, I can browse all publicly available pages from the sitemap of a Janeway site (journal/press/preprints) as if I was using the standard navigation from the front-end theme.

**context**

[WCAG 2.4.5 - Multiple Ways](https://www.w3.org/TR/WCAG22/#multiple-ways)
> More than one way is available to locate a Web page within a set of Web pages except where the Web Page is the result of, or a step in, a process.

Our themes have a navigation menu, but that is just one way. Some journals have a sitemap, but not all, and as described in https://github.com/BirkbeckCTP/hourglass/issues/247 sitemaps must include the static pages as well as the articles.  Journals have a search bar, but this only searches articles, and not the static pages or news items (e.g. [typing in 'submission' doesn't bring up the submission page ](https://www.glossa-journal.org/search/?article_search=submission&title=on&abstract=on&sort=relevance )).

Consider:
- [x] improved sitemaps
- [x] improved search
- [x] breadcrumbs

**Comment by joemull** [comment]
Discussion consensus:
- [ ] Sort out the sitemaps by adding the CMS pages.
- [ ] Update the article search page by specifying it's only searching for articles.

Improving search and considering breadcrumbs will go into phase 2 of the project.

**Comment by StephDriver** [comment]
consider also #3480 requesting article keywords as cross references.

**Comment by StephDriver** [comment]
this mostly duplicates #3919 so combining them.

the one bit missing is: https://github.com/openlibhums/janeway/issues/3919#issuecomment-2042439391

 - [ ] when updating, make sure the sitemap page has the language specified.
